src/Utils/BlockTemplateUtils.php

```diff
@@ -282,6 +282,28 @@ public static function supports_block_templates() {
 		return true;
 	}
 
+	/**
+	 * Retrieves a single unified template object using its id.
+	 *
+	 * @param string $id            Template unique identifier (example: theme_slug//template_slug).
+	 * @param string $template_type Optional. Template type: `'wp_template'` or '`wp_template_part'`.
+	 *                             Default `'wp_template'`.
+	 *
+	 * @return WP_Block_Template|null Template.
+	 */
+	public static function fse_get_block_template( $id, $template_type ) {
```

Comment:
I'm not sure about the `fse` prefix... :thinking: I would just call it `get_block_template` like in WP core or `wc_get_block_template` if we want to make it clear it's WC-specific.

Reply:
Yes, as we already discussed in our internal channel. Maybe it is better to investigate why there is this different behaviour. This can require a little bit of time :(

Reply:
I would call it `get_block_template` as well, we already have `BlockTemplateUtils` as the prefix.